sequencer/src/bin/espresso-dev-node.rs

```diff
@@ -210,15 +250,36 @@ async fn main() -> anyhow::Result<()> {
         l1_interval: _,
         max_block_size,
         epoch_height,
+        contracts,
+        l1_deployment,
     } = cli_params;
 
     logging.init();
 
-    let (l1_url, _anvil) = if let Some(url) = rpc_url {
+    let mut anvil_statefile = NamedTempFile::new()?;
```

Comment:
Should we make this configurable via CLI argument so users can choose it? How are you currently determining what file to load?

Reply:
This PR doesn't add loading into dev-node itself, it wouldn't make much difference since anvil node mines blocks on-demand anyways. We can only dump anvil state, which then can be loaded by the user when running devnode instances on their L1 instance that for whatever reason doesn't mine on-demand.

Reply:
Thanks I understand better now.